docs/api/web-request.md

```diff
@@ -68,6 +68,21 @@ The `uploadData` is an array of `UploadData` objects.
 
 The `callback` has to be called with an `response` object.
 
+Some examples of valid `urls`:
+
+```js
+'http://foo:1234/'
+'http://foo.com/'
+'http://foo:1234/bar'
+'*://*/*'
+'*://example.com/*'
+'*://example.com/foo/*'
+'http://*.foo:1234/'
+'file://foo:1234/bar'
+'http://foo:*/'
+'*://www.foo.com/'
+```
+
```

Comment:
This is a good question! In writing this, i imagined it more as a failure on our end that we were previously failing silently, in that it lead to false user assumptions and was therefore a fix although I think it could also easily and accurately be positioned as breaking change as well 🤔